skycultures/chinese_contemporary/star_names.fab

```diff
@@ -363,6 +369,7 @@
 46733|_("Inner Steps IV") 1
 43644|_("Inner Steps V") 1
 45455|_("Inner Steps VI") 1
+35783|_("Inner Steps Added I") 1
```

Comment:
Likewise, is it not possible to identify one and only one star as "Inner Steps Added I"?